config/locales/models/rdv.fr.yml

```
@@ -95,6 +95,7 @@ fr:
         rdv:
           attributes:
             base:
+              overlapping_absence: "%{agent_name} a enregistré une indisponibilité pendant la période du RDV"
```

```suggestion
              overlapping_absence: "%{agent_name} n'est pas disponible pendant la période du RDV"
```
On pourrait avoir un message d'erreur un tout petit peu moins robotique